libs/components/src/next/button/stories/vl-button.stories-arg.ts

```diff
@@ -104,6 +104,34 @@ export const buttonArgTypes: ArgTypes<ButtonArgs> = {
             defaultValue: { summary: buttonArgs.loading },
         },
     },
+    icon: {
+        name: 'icon',
+        description: 'Beeldt een icoon af in de button.<br/>Standaard wordt dit icoon voor de tekst afgebeeld.',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: buttonArgs.icon },
+        },
+    },
+    iconAfterText: {
```

Comment:
Eventueel `iconPlacement` gebruiken ipv `iconAfterText` (met dan bv. `before` als standaard waarde).

Ik geef toe dat dit weinig meerwaarde geeft en dat dit attribuut op zich ook wel duidelijk maakt waarover het gaat maar gezien we `iconPlacement` ook al voor dezelfde functionaliteit gebruiken op andere plaatsen (o.a. `vl-side-sheet`), leek het me mss goed om dit consequent door te trekken.

Alternatief, passen we `iconPlacement` aan (want op zich ook raar - had beter een boolean geweest).

Maar het is ook ok, om het zo te laten.